ext/fetch/26_fetch.js

```diff
@@ -210,44 +212,60 @@
       const reader = reqBody.getReader();
       WeakMapPrototypeSet(requestBodyReaders, req, reader);
       (async () => {
-        while (true) {
-          const { value, done } = await PromisePrototypeCatch(
-            reader.read(),
-            (err) => {
-              if (terminator.aborted) return { done: true, value: undefined };
-              throw err;
```

Comment:
I guess this throw removal is the crux of this PR?

Reply:
Yes